lib/factory_bot/definition_proxy.rb

```diff
@@ -121,6 +121,7 @@ def method_missing(name, *args, &block)
     # Except that no globally available sequence will be defined.
     def sequence(name, *args, &block)
       sequence = Sequence.new(name, *args, &block)
```

Comment:
It occurs to me that the `name` may not be important for these inline sequences. Could we just use some unique identifier as the name and register with `register_sequence`?

Thinking out loud here. What if we had an `InlineSequence` class or something like that whose `names` method just returned `[object_id]`?